scripts/e2e_compas.py

```
@@ -864,6 +1032,102 @@ def run_case(
+                compas_bin,
+                repo,
+                plugin_ids,
+                mode=install_mode,
+                registry_source=normalized_registry_source,
+                force=False,
+            )
+            summary["steps"].append({"step": "install", "status": "ok", "plugins": plugin_ids})
+
+            doctor = _plugins_doctor(compas_bin, repo, normalized_registry_source)
+            if doctor.get("ok") is not True:
+                raise RuntimeError(f"doctor failed after install: {doctor}")
```

Ruff (0.15.2)

[warning] Abstract `raise` to an inner function

(TRY301)